bika/lims/profiles/default/workflows/bika_analysis_workflow/definition.xml

```diff
@@ -42,6 +42,7 @@
       <permission-role>LabManager</permission-role>
       <permission-role>Manager</permission-role>
       <permission-role>RegulatoryInspector</permission-role>
+      <permission-role>Sampler</permission-role>
```

Comment:
Sampler mustn't be allowed to see the results set in the lab. His/her work is only about the results in the field. If there are personnel in the lab that works both as samplers and analysts, then both roles should be applied to those users